### ISSUE TITLE
Able to tweet informationes for more then one sensor

### ISSUE THREAD

**Issue by ohobby** [aschuma/air_tweets#7]
Hi,
my Suggestion is, to make this script possible to tweet this informationes for a list of sensors and not for only one sensor.

regards,
Oliver

**Comment by aschuma** [comment]
Good point.  Should a dedicated tweet be sent for each sensor violating the limit? Or do you think that a single tweet should be sent if a group of sensors exceed the limit.

**Comment by ohobby** [comment]
While looking to improve my bot, stumbled across this bot of yours again.
To the demand of then, of course for each sensor a separate tweet, because the values are exceeded differently.
Possibly the sensor IDs from a CSV, which also specifies whether the values of temperature, humidity and air pressure should be output or not.
Not every sensor has temperature, humidity and air pressure.
Example:
sensor,plz,city,district,temperatur_s,air_humidity_s,air_pressure_s
761,60386,Frankfurt,Riederwald,1,1,0
996,60389,Frankfurt,Nordend-Ost,1,2,0

0 = value is not displayed or is not present
1 = Value is displayed
2 = Value is present but incorrect (humidity)

**Comment by aschuma** [comment]
W

**Comment by aschuma** [comment]
Won't fix.
